packages/cli/package.json

```diff
@@ -22,22 +22,21 @@
     "@types/prettier": "^2.7.2",
     "@types/recursive-readdir": "^2.2.1",
     "@types/tar-fs": "^2.0.1",
-    "oclif": "3.9.1",
     "tempy": "^3.0.0",
     "type-fest": "^3.6.0",
     "vitest": "^0.33.0"
   },
   "peerDependencies": {
-    "@remix-run/react": "^1.17.1",
+    "@remix-run/react": "^1.19.1",
```

Comment:
Should we pin this dep?

Reply:
🤷🏼  our previous type compilation issues were due to an unpinned peerDep. I'd rather be safe than sorry at this point (until we can trust their internals won't change and just peer dep it all).